src/zenhub.py

```diff
@@ -110,91 +196,67 @@ def _update_issue_pipeline(self, pipeline, pos=None):
         else:
             logger.info(f'{pipeline} is not a valid pipeline.')
 
-    def _update_issue_to_epic(self):
+    def update_issue_to_epic(self):
         # Change the issue into an Epic.
-        logger.info(f'Turning {self.issue} into an epic in repo {self.repo_name}')
+        logger.info(f'Turning {self.github_key} into an epic in repo {self.github_key}')
 
-        url = os.path.join(self.url, 'convert_to_epic')
-        json_dict = {'issues': [{'repo_id': self.repo_id, 'issue_number': self.issue}]}
-        response = requests.put(url, headers=self.headers, json=json_dict)
+        json_dict = {'issues': [{'repo_id': self.repo_id, 'issue_number': self.github_key}]}
+        response = requests.post(f'{self.url}{self.repo_id}/issues/{self.github_key}/convert_to_epic', headers=self.headers, json=json_dict)
 
         if response.status_code == 200:
-            logger.info(f'Success. {self.issue} was converted to an Epic')
+            logger.info(f'Success. {self.github_key} was converted to an Epic')
         else:
             logger.info(
                 f'Error occured when updating issue to epic. Status Code: {response.status_code}. Reason: {response.reason}')
             raise RuntimeError(
                 f'Error occured when updating issue to epic. Status Code: {response.status_code}. Reason: {response.reason}')
 
-    def _get_pipeline_ids(self):
-        # Determine the valid pipeline IDs for this repo.
-        logger.info(f'Retrieving pipeline ids for {self.repo_name}.')
-        url = os.path.join(self.access_params['options']['server'], self.repo_id, 'board')
-        response = requests.get(url, headers=self.headers)
+    def update_epic_to_issue(self):
+        """Change this epic to an issue"""
+        logger.info(f'Turning {self.github_key} into an issue in repo {self.github_repo}')
+
+        json_dict = {'issues': [{'repo_id': self.repo_id, 'issue_number': self.github_key}]}
+        print(f'{self.url}{self.repo_id}/epics/{self.github_key}/convert_to_issue')
+        response = requests.put(f'{self.url}{self.repo_id}/epics/{self.github_key}/convert_to_issue', headers=self.headers, json=json_dict)
 
         if response.status_code == 200:
-            logger.info(f'Successfully retrieved pipeline ids for {self.repo_name}.')
-            data = response.json()
-            ids = {pipeline['name']: pipeline['id'] for pipeline in data['pipelines']}
-            return ids
+            logger.info(f'Success. {self.github_key} was converted to an Epic')
         else:
             logger.info(
```

Comment:
I think we want this to be `logger.debug`

Reply:
done